src/praisonai-agents/tests/unit/test_tool_result_guardrail.py

```
@@ -0,0 +1,126 @@
+
+class TestProtocolSurface:
+    def test_method_is_part_of_protocol_surface(self):
+        assert "validate_tool_result" in GUARDRAIL_PROTOCOL_METHODS
+
+    def test_result_only_object_is_a_guardrail(self):
+        assert is_guardrail_object(RedactingGuardrail()) is True
+
+
+class TestGuardrailChain:
+    def test_chain_rejects(self):
+        ok, out = GuardrailChain([NoSecretsGuardrail()]).validate_tool_result(
```

Ruff (0.16.3)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)